documentation-site/examples/app-nav-bar/basic.js

```diff
@@ -34,6 +37,7 @@ const MAIN_NAV = [
     mapItemToNode: renderItem,
     mapItemToString: renderItem,
     navExitIcon: Delete,
+    navPosition: {desktop: NAV_POSITION.horizontal},
```

Comment:
following my previous comment,  should it be just `position`?

Reply:
Since it relates to the sub-navigation positioning and not the item positioning itself I'd keep it as `navPosition` in this case.

Reply:
oh, I see, at first, it was not clear for me - in that case, should name it `subNavPosition`?

Reply:
Since I use `nav`|`navExitIcon` thi one would be `navPosition` to match. I'm open to renaming all of the mentioned to `subnav`|`subnavExitIcon`|`subnavPosition` if you think it would reflect it's purpose better, but I personally like `nav` prefix so it's like "navigation under current item regardless how deep it's nested".

Reply:
my thought process was that if it applies to the subnav, probably we should clarify, but I don't feel strongly either way

maybe a fresh pair of eyes could help? cc @chasestarr @tajo @sandgraham

Reply:
my comment from the last pr that suggested `navPosition` https://github.com/uber/baseweb/pull/3166#discussion_r406465811